src/bsoncxx/types.hpp

```diff
@@ -309,6 +314,12 @@ struct BSONCXX_API b_date {
         : value(std::chrono::duration_cast<std::chrono::milliseconds>(tp.time_since_epoch())) {
     }
 
+    ///
+    /// Default constructor for b_date.
+    ///
+    BSONCXX_INLINE b_date() : value(0) {
```

Comment:
I just tested this and it seems like the default constructor for `std::chrono::milliseconds` doesn't set it to 0, so I'm leaving this as is.